core/modules/dashboard/dashboard.module

```diff
@@ -0,0 +1,279 @@
+<?php
+/**
+ * @file
+ * Provides an administrative dashboard.
+ */
+
+/**
+ * Implements hook_menu().
+ */
+function dashboard_menu() {
+  $items = array();
+
+  // Check if any dashboard is present and is not disabled.
+  $dashboard_layouts = layout_load_multiple_by_path('admin/dashboard');
+  $dashboard_layout = array_pop($dashboard_layouts);
+  if ($dashboard_layout->hasContexts(array('dashboard')) && !$dashboard_layout->disabled) {
+    $items['admin/dashboard/overview'] = array(
+      'title' => 'Overview',
+      'type' => MENU_DEFAULT_LOCAL_TASK,
+      'weight' => -100
+    );
+    $items['admin/dashboard/settings'] = array(
+      'title' => 'Settings',
+      'page callback' => 'backdrop_get_form',
+      'page arguments' => array('dashboard_admin_settings'),
+      'description' => 'Admin dashboard settings',
+      'access arguments' => array('administer dashboard'),
+      'type' => MENU_LOCAL_TASK,
+      'weight' => -10,
+    );
+  }
+
+  return $items;
+}
+
+/**
+ * Implements hook_menu().
+ */
+function dashboard_menu_alter(&$items) {
+  if (isset($items['admin/dashboard'])) {
+    // Add a description for the admin and admin/index pages.
+    $items['admin/dashboard']['description'] = 'Get an overview of your site and manage administrative tasks.';
+  }
+}
+
+/**
+ * Implements hook_permission().
+ */
+function dashboard_permission() {
+  return array(
+    'access dashboard' => array(
+      'title' => t('Access Dashboard'),
+      'description' => t('Access the dashboard.'),
+    ),
+    'administer dashboard' => array(
+      'title' => t('Administer Dashboard'),
+      'description' => t('Change the dashboard settings, such as redirecting behavior on login.'),
+    ),
+  );
+}
+
+/**
+ * Implements hook_config_info().
+ */
+function dashboard_config_info() {
+  $prefixes['layout.layout.dashboard'] = array(
+    'label' => t('Dashboard default layout'),
+    'group' => t('Layouts'),
+  );
+  $prefixes['layout.menu_item.dashboard'] = array(
+    'label' => t('Dashboard default menu item'),
+    'group' => t('Layout menu items'),
+  );
```

Comment:
I'm not sure if these should be added. They're already claimed by `layout_config_info()` and we don't do this for default views that are shipped with modules.

Reply:
ok. I thought we had to do this to get them removed automatically on uninstall? Byt since we are doing it manually anyway, I'll kill these.